opennlp-tools/src/main/java/opennlp/tools/postag/TagDictionary.java

```diff
@@ -25,11 +25,18 @@
 public interface TagDictionary {
 
   /**
-   * Returns a list of valid tags for the specified word.
+   * Retrieves a list of valid tags for the specified {@code word}.
    *
    * @param word The word.
-   * @return A list of valid tags for the specified word or null if no information
-   * is available for that word.
+   * @return An array of valid tags for the specified {@code word} or {@code null} if
+   *         no information is available for that word.
    */
   String[] getTags(String word);
+
+  /**
+   * Whether if the dictionary is case-sensitive or not.
```

Comment:
I think we can drop the `if` (i.e. Whether the dictionary is... or not).

Reply:
Did not spot that one, or misread the sentence in that moment.